packages/server/postgres/queries/getMeetingTaskEstimates.ts

```diff
@@ -9,11 +9,8 @@ export interface MeetingTaskEstimatesResult extends IGetMeetingTaskEstimatesQuer
   stageId: string
 }
 
-const getMeetingTaskEstimates = async (
-  taskIds: MaybeReadonly<string[]>,
-  meetingIds: MaybeReadonly<string[]>
-) => {
-  return getLatestTaskEstimatesQuery.run({taskIds, meetingIds} as any, getPg()) as Promise<
+const getMeetingTaskEstimates = async (taskIds: MaybeReadonly<string[]>) => {
```

Comment:
Is this change of removing `meetingIds` from the argument list expected?

Reply:
Yes, if you take a look at the underlying query it doesn't have `meetingIds` and after upgrading `pgtyped` TS complained
```
 Object literal may only specify known properties, and 'meetingIds' does not exist in type 'IGetLatestTaskEstimatesQueryParams'.ts(2345)
```